utils/keeper-bench/Generator.h

```diff
@@ -122,5 +122,19 @@ class SetRequestGenerator final : public IGenerator
     uint64_t data_size;
 };
 
+class MixedRequestGenerator final : public IGenerator
```

Comment:
This is an adhoc solution but I think it would be nice to make it customizable so that we can define any combination of other generators with specified ratios.